src/Apps/Order/Routes/Accept/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { TransactionDetailsSummaryItemFragmentContainer as TransactionDetailsSummaryItem } from "Apps/Order/Components/TransactionDetailsSummaryItem"
+import { CountdownTimer } from "Styleguide/Components/CountdownTimer"
+import { get } from "Utils/get"
+import { ArtworkSummaryItemFragmentContainer as ArtworkSummaryItem } from "../../Components/ArtworkSummaryItem"
+import { CreditCardSummaryItemFragmentContainer as CreditCardSummaryItem } from "../../Components/CreditCardSummaryItem"
+
+interface AcceptProps {
+  mediator: Mediator
+  order: Accept_order
+  relay?: RelayProp
+  router: Router
+  route: RouteConfig
+}
+
+interface AcceptState {
+  isCommittingMutation: boolean
+}
+
+export class Accept extends Component<AcceptProps, AcceptState> {
+  state = {
+    isCommittingMutation: false,
+  }
+
+  onSubmit: () => void = () => {
+    console.log("SUBMITTED!")
+  }
+
+  render() {
+    const { order } = this.props
+    console.log("ORDER", order)
+    const { isCommittingMutation } = this.state
+    const artwork = get(
+      this.props,
+      props => order.lineItems.edges[0].node.artwork
+    )
+
+    return (
+      <>
+        <HorizontalPadding px={[0, 4]}>
+          <Row>
+            <Col>
+              <OrderStepper
+                currentStep="Review"
+                steps={counterofferFlowSteps}
+              />
+            </Col>
+          </Row>
+        </HorizontalPadding>
+        <HorizontalPadding>
+          <TwoColumnLayout
+            Content={
+              <Flex
+                flexDirection="column"
+                style={isCommittingMutation ? { pointerEvents: "none" } : {}}
+              >
+                <Media at="xs">
+                  <ArtworkSummaryItem order={order} />
+                </Media>
+                <Media at="xs">
+                  <Spacer mb={2} />
+                </Media>
+                <Flex flexDirection="column">
+                  <CountdownTimer
+                    action="Respond"
+                    note="Expired offers end the negotiation process permanently."
+                    countdownStart={order.lastOffer.createdAt}
+                    countdownEnd={order.stateExpiresAt}
+                  />
+                  <TransactionDetailsSummaryItem
+                    order={order}
+                    renderHeaderEntry={this.renderHeaderEntry.bind(this)}
+                  />
+                </Flex>
+                <Spacer mb={[2, 3]} />
+                <Media greaterThan="xs">
+                  <Button
+                    onClick={this.onSubmit}
+                    loading={isCommittingMutation}
+                    size="large"
+                    width="100%"
+                  >
+                    Submit
+                  </Button>
+                  <Spacer mb={2} />
+                  <Sans textAlign="center" size="2" color="black60">
```

Comment:
I put a note on @ds300 's PR as well, but at some point we should probably extract this little component since we keep repeating it

Reply:
That was done and is now merged in. It's called `ConditionsOfSaleDisclaimer`